src/ring_theory/power_series/basic.lean

```diff
@@ -969,6 +971,45 @@ lemma is_unit_constant_coeff (φ : power_series R) (h : is_unit φ) :
   is_unit (constant_coeff R φ) :=
 mv_power_series.is_unit_constant_coeff φ h
 
+open finset nat
+
+variables {A : Type*} [comm_ring A]
+
+/-- The ring homomorphism taking a power series `f(X)` to `f(aX)`. -/
+noncomputable def eval_mul_hom (a : A) : power_series A →+* power_series A :=
```

Comment:
This should move down into the `section comm_semiring` below, and use `R` instead of `A`.

Reply:
Done, thank you!